pkg/oci/utils.go

```diff
@@ -218,3 +218,38 @@ func stateToOCIState(state vc.State) string {
 		return ""
 	}
 }
+
+// EnvVars converts an OCI process environment variables slice
+// into a virtcontainers EnvVar slice.
+func EnvVars(envs []string) ([]vc.EnvVar, error) {
+	var envVars []vc.EnvVar
+
+	envDelimiter := "="
+	expectedEnvLen := 2
+
+	for _, env := range envs {
+		envSlice := strings.Split(env, envDelimiter)
```

Comment:
This needs to be something like:

```
strings.Split(env, envDelimiter, expectedEnvLen)

if len(envSlice) < expectedEnvLen {
    // ...
}
```

Otherwise, values like `foo=bar=` will be rejected incorrectly.